Library/Formula/sassc.rb

```diff
@@ -2,7 +2,7 @@
 
 class Sassc < Formula
   homepage "https://github.com/sass/sassc"
-  url "https://github.com/sass/sassc/archive/3.1.0.tar.gz"
+  url "https://github.com/sass/sassc/archive/3.2.0.tar.gz"
   sha1 "573224ad922b46ea1a568807ddcbc7de41a4254d"
```

Comment:
Yup, that's the one :+1:. Just replace the `sha1` in the formula with the new `sha256`.

```diff
- sha1 "573224ad922b46ea1a568807ddcbc7de41a4254d"
+ sha256 "73454e810aba782466df4dcf59e73142bece37f494776d8bab2e3dae88992a24"
```